### DISCUSSION TITLE
Main screen layout improvement when cameras number not even

### DISCUSSION THREAD

**Discussion by potat0man** [blakeblackshear/frigate#11669] [Ideas]
When cameras number not even and one camera stays in bottom corner.
Maybe improvement layout would be one main camera and all the rest below:
<img width="539" alt="image" src="https://github.com/blakeblackshear/frigate/assets/67843546/f788f265-b273-4172-a28b-fd6ac14f5740">

**Comment by NickM-27** [comment]
this is not very simple to calculate on the fly. I would suggest just making a camera group and making your own layout if that is how you want it to look
